error: no viable conversion from 'time_point<std::__1::chrono::steady_clock, duration<[...], ratio<[...], 1000000000>>>' to 'const time_point<std::__1::chrono::system_clock, duration<[...], ratio<[...], 1000000>

b0f73017e2fc0563331e76573df1ce08571c1a80 breaks on macOS 10.15:

```
     110    FAILED: fairmq/CMakeFiles/fairmq-shmmonitor.dir/shmem/Monitor.cxx.o
     111    /Users/alfaci/alfaci/workspace/FairRootGroup_FairSoft_PR-284/spack/
            lib/spack/env/clang/clang++  -DBOOST_ALL_NO_LIB -DBOOST_DATE_TIME_D
            YN_LINK -DBOOST_ERROR_CODE_HEADER_ONLY -DBOOST_PROGRAM_OPTIONS_DYN_
            LINK -I/Users/alfaci/alfaci/workspace/FairRootGroup_FairSoft_PR-284
            /build/fairsoft_ctest/stage/spack-stage-fairmq-develop-7tsrcrzovgt6
            e4qyz3mdoayuwlfhflgg/spack-src -isystem /Users/alfaci/alfaci/worksp
            ace/FairRootGroup_FairSoft_PR-284/build/fairsoft_ctest/install-tree
            /darwin-catalina-x86_64/clang-11.0.3-apple/boost-1.70.0-wm727ktd4bi
            vhwzfry6wvu7t4gq2uwyy/include -isystem /Users/alfaci/alfaci/workspa
            ce/FairRootGroup_FairSoft_PR-284/build/fairsoft_ctest/stage/spack-s
            tage-fairmq-develop-7tsrcrzovgt6e4qyz3mdoayuwlfhflgg/spack-src/exte
            rn/PicoSHA2 -O2 -g -Wshadow -Wall -Wextra -Wpedantic -DNDEBUG -isys
            root /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.pl
            atform/Developer/SDKs/MacOSX10.15.sdk -fPIE   -std=c++14 -MD -MT fa
            irmq/CMakeFiles/fairmq-shmmonitor.dir/shmem/Monitor.cxx.o -MF fairm
            q/CMakeFiles/fairmq-shmmonitor.dir/shmem/Monitor.cxx.o.d -o fairmq/
            CMakeFiles/fairmq-shmmonitor.dir/shmem/Monitor.cxx.o -c /Users/alfa
            ci/alfaci/workspace/FairRootGroup_FairSoft_PR-284/build/fairsoft_ct
            est/stage/spack-stage-fairmq-develop-7tsrcrzovgt6e4qyz3mdoayuwlfhfl
            gg/spack-src/fairmq/shmem/Monitor.cxx
  >> 112    /Users/alfaci/alfaci/workspace/FairRootGroup_FairSoft_PR-284/build/
            fairsoft_ctest/stage/spack-stage-fairmq-develop-7tsrcrzovgt6e4qyz3m
            doayuwlfhflgg/spack-src/fairmq/shmem/Monitor.cxx:312:62: error: no 
            viable conversion from 'time_point<std::__1::chrono::steady_clock, 
            duration<[...], ratio<[...], 1000000000>>>' to 'const time_point<st
            d::__1::chrono::system_clock, duration<[...], ratio<[...], 1000000>
            >>'
     113                time_t current = chrono::system_clock::to_time_t(now);
     114                                                                 ^~~
     115    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.
            xctoolchain/usr/bin/../include/c++/v1/chrono:1325:28: note: candida
            te constructor (the implicit copy constructor) not viable: no known
             conversion from 'std::__1::chrono::time_point<std::__1::chrono::st
            eady_clock, std::__1::chrono::duration<long long, std::__1::ratio<1
            , 1000000000> > >' to 'const std::__1::chrono::time_point<std::__1:
            :chrono::system_clock, std::__1::chrono::duration<long long, std::_
            _1::ratio<1, 1000000> > > &' for 1st argument
     116    class _LIBCPP_TEMPLATE_VIS time_point
     117                               ^
     118    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.
            xctoolchain/usr/bin/../include/c++/v1/chrono:1325:28: note: candida
            te constructor (the implicit move constructor) not viable: no known
             conversion from 'std::__1::chrono::time_point<std::__1::chrono::st
            eady_clock, std::__1::chrono::duration<long long, std::__1::ratio<1
            , 1000000000> > >' to 'std::__1::chrono::time_point<std::__1::chron
            o::system_clock, std::__1::chrono::duration<long long, std::__1::ra
            tio<1, 1000000> > > &&' for 1st argument
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -94,7 +94,7 @@ endif()
 
 if(BUILD_FAIRMQ OR BUILD_SDK)
   find_package2(PUBLIC FairLogger REQUIRED
-    VERSION 1.2.0
+    VERSION 1.6.0
   )
 
   foreach(dep IN LISTS FairLogger_PACKAGE_DEPENDENCIES)

--- a/fairmq/CMakeLists.txt
+++ b/fairmq/CMakeLists.txt
@@ -380,6 +380,7 @@ if(BUILD_FAIRMQ)
     Boost::boost
     Boost::date_time
     Boost::program_options
+    FairLogger::FairLogger
     PicoSHA2
   )
   target_include_directories(fairmq-shmmonitor PUBLIC

--- a/fairmq/shmem/Monitor.cxx
+++ b/fairmq/shmem/Monitor.cxx
@@ -10,6 +10,7 @@
 #include "Common.h"
 
 #include <fairmq/Tools.h>
+#include <fairlogger/Logger.h>
 
 #include <boost/interprocess/managed_shared_memory.hpp>
 #include <boost/interprocess/file_mapping.hpp>
@@ -79,6 +80,10 @@ Monitor::Monitor(const string& shmId, bool selfDestruct, bool interactive, bool 
             throw DaemonPresent(tools::ToString("fairmq-shmmonitor for shared memory id ", fShmId, " already started or not properly exited."));
         }
     }
+
+    Logger::SetConsoleColor(false);
+    Logger::DefineVerbosity(Verbosity::user1, VerbositySpec::Make(VerbositySpec::Info::timestamp_us));
+    Logger::SetVerbosity(Verbosity::verylow);
 }
 
 void Monitor::CatchSignals()
@@ -309,17 +314,10 @@ void Monitor::CheckSegment()
                  << setw(10) << (fViewOnly ? "view only" : to_string(duration)) << " |"
                  << c << flush;
         } else if (fViewOnly) {
-            time_t current = chrono::system_clock::to_time_t(now);
-            chrono::milliseconds ms = chrono::duration_cast<chrono::milliseconds>(now.time_since_epoch());
-            struct tm local;
-            localtime_r(&current, &local);
-            char timeBuffer[80];
-            size_t count = strftime(timeBuffer, 80, "%F %T", &local);
-            cout << (count != 0 ? timeBuffer : "") << "." << setfill('0') << setw(6) << ms.count() % 1000000
-                 << ", name: " << fSegmentName
-                 << ", size: " << segment.get_size()
-                 << ", free: " << segment.get_free_memory()
-                 << ", numDevices: " << numDevices << endl;
+            LOGV(info, user1) << "name: " << fSegmentName
+                                          << ", size: " << segment.get_size()
+                                          << ", free: " << segment.get_free_memory()
+                                          << ", numDevices: " << numDevices;
         }
     } catch (bie&) {
         fHeartbeatTriggered = false;

--- a/fairmq/shmem/Monitor.cxx
+++ b/fairmq/shmem/Monitor.cxx
@@ -314,10 +314,14 @@ void Monitor::CheckSegment()
                  << setw(10) << (fViewOnly ? "view only" : to_string(duration)) << " |"
                  << c << flush;
         } else if (fViewOnly) {
-            LOGV(info, user1) << "name: " << fSegmentName
-                                          << ", size: " << segment.get_size()
-                                          << ", free: " << segment.get_free_memory()
-                                          << ", numDevices: " << numDevices;
+            size_t free = segment.get_free_memory();
+            size_t total = segment.get_size();
+            size_t used = total - free;
+            LOGV(info, user1) << "[" << fSegmentName
+                              << "] devices: " << numDevices
+                              << ", total: " << total
+                              << ", free: " << free
+                              << ", used: " << used;
         }
     } catch (bie&) {
         fHeartbeatTriggered = false;

--- a/fairmq/shmem/runMonitor.cxx
+++ b/fairmq/shmem/runMonitor.cxx
@@ -118,6 +118,9 @@ int main(int argc, char** argv)
         }
 
         cout << "Starting shared memory monitor for session: \"" << sessionName << "\" (shmId: " << shmId << ")..." << endl;
+        if (viewOnly && !interactive) {
+            cout << "running in non-interactive view-only mode, outputting with interval of " << intervalInMS << "ms. (change with --interval), press ctrl+C to exit." << endl;
+        }
 
         Monitor monitor(shmId, selfDestruct, interactive, viewOnly, timeoutInMS, intervalInMS, runAsDaemon, cleanOnExit);
 

